src/validate.py

```diff
@@ -36,6 +39,10 @@
     raise ValueError(
         "You need to set HF_TOKEN to download some gated model from HuggingFace"
     )
+CLEAN_MODELS_CACHE = os.getenv("CLEAN_MODELS_CACHE")
```

Comment:
Sorry I didn't make this clear enough previsouly. We should make this as an flag to `loop` command.
something like `--auto_clean_cache`